browser_patches/webkit/patches/bootstrap.diff

```diff
@@ -13242,7 +14333,17 @@ index 0000000000000000000000000000000000000000..4ed6100fed7ed407c3135e768dea6d9f
 +        eventClickCount,
 +        eventModifiers,
 +        timestamp);
++#if PLATFORM(WPE) || PLATFORM(GTK)
++    // We intercept any drags generated by this mouse event
++    // to prevent them from creating actual drags in the host
++    // operating system.
++    // So far this is only implemented for WPE.
```

Comment:
does GTK code not work (I see it is enabled)?

Reply:
stale comment, fixed!